### PR TITLE
[Docs/JSON-RPC] Fix suix_getCoinMetadata, suix_getStakes examples

### PR DESCRIPTION
**Pull request by amnn**
## Description

`suix_getCoinMetadata` was using the internal Rust `CoinMetadata` type in the example, rather than the type actually used by JSON RPC (`SuiCoinMetadata`).

`suix_getStakes` returns a list, rather than a single delegated stake structure.

## Test Plan

Make calls to the relevant APIs in fullnode, and make sure the output matches the example:

```shell
$ curl -LX POST  "https:/fullnode.mainnet.sui.io:443" \
    --header 'Content-Type: application/json'     \
    --data-raw '{
    "jsonrpc": "2.0",
    "method": "suix_getCoinMetadata",
    "id": 1,
    "params": ["0x2::sui::SUI"]
}' | jq  -C .

$ curl -LX POST  "https:/fullnode.mainnet.sui.io:443" \
    --header 'Content-Type: application/json' \
    --data-raw '{
    "jsonrpc": "2.0",
    "method": "suix_getStakes",
    "id": 1,
    "params": ["'$ADDRESS'"]
}' | jq  -C .
```

### CODE REVIEW

File: crates/sui-open-rpc/spec/openrpc.json [hunk 1]
```diff
@@ -3744,34 +3742,42 @@
...
-                {
-                  "stakedSuiId": "0x02c75973a51c17180798237326a58694a2cf5cd6fa76ed1d18f05f15e3507525",
-                  "stakeRequestEpoch": "244",
-                  "stakeActiveEpoch": "245",
-                  "principal": "200000000000",
-                  "status": "Unstaked"
-                }
-              ]
-            }
+            "value": [
```
Comment (amnn) [inline crates/sui-open-rpc/spec/openrpc.json:3745]: This is one of the main changes -- wrapping the result in a list, outermost.

File: crates/sui-open-rpc/spec/openrpc.json [hunk 2]
```diff
@@ -3151,14 +3151,12 @@
...
-              "id": {
-                "id": "0x6d907beaa3a49db57bdfdb3557e6d405cbf01c293a53e01457d65e92b5d8dd68"
-              },
               "decimals": 9,
               "name": "Usdc",
               "symbol": "USDC",
               "description": "Stable coin.",
-              "icon_url": null
+              "iconUrl": null,
+              "id": "0x6d907beaa3a49db57bdfdb3557e6d405cbf01c293a53e01457d65e92b5d8dd68"
```
Comment (amnn) [inline crates/sui-open-rpc/spec/openrpc.json:3159]: This is the other meaningful change -- changing how the `id` is represented and typo on `iconUrl`.

**Review by amnn — COMMENTED** [review]
Marking relevant changes in examples.

File: crates/sui-open-rpc/src/examples.rs [hunk 3]
```diff
@@ -1406,35 +1399,44 @@ impl RpcExampleProvider {
...
+                validator_address: SuiAddress::from(ObjectID::new(self.rng.gen())),
+                staking_pool: ObjectID::new(self.rng.gen()),
+                stakes: vec![
+                    Stake {
+                        staked_sui_id: ObjectID::new(self.rng.gen()),
+                        stake_request_epoch: 62,
+                        stake_active_epoch: 63,
+                        principal,
+                        status: StakeStatus::Active {
+                            estimated_reward: (principal as f64 * 0.0026) as u64,
```
Comment (stefan-mysten) [inline crates/sui-open-rpc/src/examples.rs:1413]: Confused about this estimated_reward, is it just a random value?

Comment (amnn) [reply crates/sui-open-rpc/src/examples.rs:1413]: Yes, I think so -- these are just made up examples.

**Pull request merged**